snap/backend/src/config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
+	statisticsServiceURL: {
+		mainnet: 'https://symbol.services',
+		testnet: 'https://testnet.symbol.services'
```

Comment:
does nodewatch works for this? - https://nodewatch.symbol.tools/api/symbol/nodes/peer

Reply:
It's not a requirement for this PR, but something you should think about.  Any feature lacking from NodeWatch please post in discord for us to improve on 🙏🏾

Reply:
Sure 👍🏼